earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/globe_cutter_app.py

```diff
@@ -512,12 +547,11 @@ def BuildSearchDb(self, source, polygon):
         try:
           self.Status("Querying search poi data: poi_id=%s, polygon=%s" %
                       (poi_id, polygon))
-          fp = urllib.urlopen(url)
-          http_status_code = fp.getcode()
+          data, http_status_code = self.HttpGet(url)
           if http_status_code == 200:
             self.Status("Copying search poi data: gepoi_%s to globe" % poi_id)
             fpw = open(search_file, "w")
-            fpw.write(fp.read().strip())
+            fpw.write(data.strip())
             fpw.write("\n")
             fpw.close()
```

Comment:
I know this isn't part of your change but since you are changing things close but see if we can add a `with` here too.